chains/fraxtal/metadata.yaml

```diff
@@ -15,6 +15,7 @@ deployer:
 displayName: Fraxtal
 domainId: 252
 gasCurrencyCoinGeckoId: frax-ether
+gnosisSafeTransactionServiceUrl: https://safe.origin.mainnet.frax.com/txs
```

Comment:
this is expected to give issues right? maybe a comment indicating such would be nice